src/code/z_sram.c

```diff
@@ -376,28 +376,28 @@ void Sram_InitDebugSave(void) {
         }
     }
 
-    gSaveContext.entranceIndex = ENTR_SPOT00_0;
+    gSaveContext.entranceIndex = ENTR_HYRULE_FIELD_0;
     gSaveContext.magicLevel = 0;
     gSaveContext.sceneFlags[5].swch = 0x40000000;
```

Comment:
```suggestion
    gSaveContext.sceneFlags[SCENE_WATER_TEMPLE].swch = 0x40000000;
```
Any chance you could you slide this in here? (And also another `sceneFlags[5]` in sram)

Reply:
I'd like to keep this PR a gross replace-all, it's easier to maintain

I plan to do manual cleanup on various things later, will add this to the list

If we really want it now though, I can do the cleanup right away